### DISCUSSION TITLE
InputNumber 中addonBefore、addonAfter不生效问题

### DISCUSSION THREAD

**Discussion by rookienook** [ant-design/ant-design#33141] [Q&A]
在使用InputNumber时，发现addonBefore、addonAfter不生效，而且控制台中报错，部分报错如下
 React does not recognize the `addonAfter` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `addonafter` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
我于是查找了代码，发现无论是antd中的`InputNumberProps`还是rc-input-number的`InputNumberProps`都没有`addonBefore`、`addonAfter`字段，但是为什么antd组件库的InputNumber组件可以使用`addonBefore`、`addonAfter`？有没有大佬帮我解答一下？

**Comment by rookienook** [comment]
我的版本是antd5.0，react版本17.0.0

**Reply by kaoding** [reply]
5.0还没有被出来，所以你就会报错。

**Reply by rookienook** [reply]
不好意思说错了ant-design-pro5.0，antd4.16.13,react版本17.0.0

**Comment by kaoding** [comment]
![image](https://user-images.githubusercontent.com/18542211/145132820-2daadb30-2937-4185-8310-9d4ac06b5b65.png)

**Reply by rookienook** [reply]
非常感谢
